web/src/components/core/IssuesPage.jsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useCallback, useEffect, useState } from "react";
+
+import { HelperText, HelperTextItem, Skeleton } from "@patternfly/react-core";
+
+import { partition, useCancellablePromise } from "~/utils";
+import { If, Page, Section } from "~/components/core";
+import { Icon } from "~/components/layout";
+import { useInstallerClient } from "~/context/installer";
+import { useNotification } from "~/context/notification";
+
+/**
+ * Renders an issue
+ * @component
+ *
+ * @param {object} props
+ * @param {import ("~/client/mixins").Issue} props.issue
+ */
+const IssueItem = ({ issue }) => {
+  const variant = issue.severity === "warn" ? "warning" : "error";
+  const icon = issue.severity === "warn" ? "warning" : "error";
```

Comment:
NP: they look like a duplicated line. Maybe we can go for an `severity`, `type`, `issueType`, or similar and use it in both, _variant_ and _name_ props.

Reply:
Well, I was also thinking about it when I wrote that lines. But having both variant and icon name with exactly the very same value ("warning" or "error") was only a great coincidence. For example, if we want to use another icon for errors, then we need to split it in two lines again.